doc/api/modules.md

```diff
@@ -1033,6 +1033,85 @@ import('fs').then((esmFS) => {
 });
 ```
 
+## Source Map V3 Support
+
+<!--introduced_in=REPLACEME-->
```

Comment:
I know it’s not ideal, but the `introduced_in` tags only work for individual pages (i.e. `.md` files); for headings, using the standard `added: REPLACME` format is better.

(Honestly, we should probably replace `introduced_in` by the YAML format.)